x-pack/plugins/security_solution/common/detection_engine/schemas/request/create_rules_schema.test.ts

```diff
@@ -1660,5 +1660,84 @@ describe('create rules schema', () => {
       };
       expect(message.schema).toEqual(expected);
     });
+
+    describe('threat_mapping', () => {
+      test('You can set a threat query, index, mapping, filters on a pre-packaged rule', () => {
```

Comment:
I think it should be:
```suggestion
      test('You can set a threat query, index, mapping, filters when creating rule', () => {
```

Reply:
Yep, good catch, fixed for the next commit